channels/management/commands/runserver.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import threading
 
-from daphne.server import Server
+from daphne.server import Server, build_endpoint_description_strings
```

Comment:
That import is giving me problems when I try to run:

python manage.py runserver

I get the following error:

ImportError: cannot import name 'build_endpoint_description_strings'

Do you happen to know what could be the problem for this?